arangod/Cache/Manager.cpp

```diff
@@ -621,7 +649,7 @@ ErrorCode Manager::rebalance(bool onlyCalculate) {
 
 void Manager::shrinkOvergrownCaches(Manager::TaskEnvironment environment) {
   TRI_ASSERT(_lock.isLockedWrite());
-  for (auto it : _caches) {
+  for (auto& it : _caches) {
     std::shared_ptr<Cache>& cache = it.second;
```

Comment:
NIT 
maybe to make it more clear
```suggestion
  for (auto& [_, cache] : _caches) {
```

Reply:
Done